Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM node:10
+# Use alpine node version
+FROM node:10.15.3-alpine
```

Comment:
Changed to use the alpine version of node as it's much smaller than the full image.  Also matched the version to the demo services for consistency.